### DISCUSSION TITLE
No output folder created; run.sh ends with error. Help would be appreciated

### DISCUSSION THREAD

**Discussion by DioCoN-PRiMe** [LSPosed/MagiskOnWSALocal#644] [Q&A]
I ran "run.sh", everything was fine until it reached the following:

            Convert vhdx to RAW image
            Convert vhdx to RAW image done
            
            Mount images
            fuse.erofs 1.6-g689372cd
            disk: /tmp/wsa-build-a92Zd0QPEN_/wsa/x64/system.img
            offset: 0
            mountpoint: /tmp/wsa-build-a92Zd0QPEN_/erofs
            dbglevel: 0
            <E> erofs: cannot find valid erofs superblock
            failed to read erofs super block
            Build: an error has occurred, exit
            
            Cleanup Work Directory
            Cleanup Mount Directory
            umount: /tmp/wsa-build-a92Zd0QPEN_/erofs/vendor: no mount point specified.
            umount: /tmp/wsa-build-a92Zd0QPEN_/erofs/system_ext: no mount point specified.
            umount: /tmp/wsa-build-a92Zd0QPEN_/erofs/product: no mount point specified.
            umount: /tmp/wsa-build-a92Zd0QPEN_/erofs: not mounted.
            umount: /tmp/wsa-build-a92Zd0QPEN_/system_root_merged/vendor: no mount point specified.
            umount: /tmp/wsa-build-a92Zd0QPEN_/system_root_merged/system_ext: no mount point specified.
            umount: /tmp/wsa-build-a92Zd0QPEN_/system_root_merged/product: no mount point specified.
            umount: /tmp/wsa-build-a92Zd0QPEN_/system_root_merged: no mount point specified.
            Cleanup Download Directory
            
            
Any help would be greatly appreciated.

**Comment by toadicusrex** [comment]
I've got the same problem

**Comment by cchsieh** [comment]
check https://github.com/LSPosed/MagiskOnWSALocal/discussions/630

modify build.sh ( https://github.com/LSPosed/MagiskOnWSALocal/pull/620/files ) and working

**Comment by DioCoN-PRiMe** [comment]
Thanks! I'll try that and post the result.
